### ISSUE TITLE
Curly Quotes Check

### ISSUE THREAD

**Issue by okrick** [DistributedProofreaders/guiguts-py#627]
![Capture](https://github.com/user-attachments/assets/fe53e3b5-ed77-419b-bc5d-d1ae66eb19de)

I would have liked to see the option to insert either prime `′` or double `″` prime marks in the `Insert:` menu, along with the left and right double and single quotation marks.  Thus, I could easily have changed the `34° 30[']` to `34° 30′`.

**Comment by windymilla** [comment]
Thanks, @okrick.

To be honest, I think primes and other less-common quotes would be better dealt with by a search & replace before running Curly Quote conversion. If all primes are marked up by the proofers, as you have them, then a simple S&R would fix them all in one go.

There were a couple of reasons I was a bit reluctant to add single & double primes (and other quotes that I considered). The main one was that many of the different quotes do look very similar to each other, and I could see a PPer accidentally inserting the wrong one and not realizing. The other was the general busyness of the dialog. Although in your screenshot you have it wide, if the dialog is narrower with a few more "insert" buttons, then the left hand action buttons would clash with the right hand insert buttons.

So, in summary, I'd prefer not to do this, unless I've missed something in my understanding of the process.

**Comment by okrick** [comment]
I'm OK with closing this one.
